import-export/to-sheets.js

```diff
@@ -56,7 +56,7 @@ const saveEntriesToSheets = (entries, entryFields) => {
                 const transform = field.sheet.toSheet || (d => d)
 
                 //  For fields that accept an array of values
-                if (field.isArrayOfValues()) {
+                if (Array.isArray(value)) {
```

Comment:
@raagavi158 what exactly does this change? Shouldn't isArrayOfValues() do the same thing -- see https://github.com/cestastanford/grandtour/blob/61a41006f70099412bfeefc8bbcf8be8db88c388/models/entry-fields/index.js#L73 ?

Reply:
@epicfaace it technically should but for whatever reason 'field' wasn't passing that check but 'value' passes the check. Changing that line fixed the probelm

Reply:
Ah I see! Didn't notice that difference before.